internal/client.go

```diff
@@ -80,14 +104,12 @@ type (
 		//	- InternalServiceError
 		TerminateWorkflow(ctx context.Context, workflowID string, runID string, reason string, details []byte) error
 
-		// GetWorkflowHistory gets history of a particular workflow.
+		// GetWorkflowHistory gets the point in time snapshot of history of a particular workflow
 		// - workflow ID of the workflow.
-		// - runID can be default(empty string). if empty string then it will pick the running execution of that workflow ID.
-		// The errors it can return:
-		//	- EntityNotExistsError
-		//	- BadRequestError
-		//	- InternalServiceError
-		GetWorkflowHistory(ctx context.Context, workflowID string, runID string) (*s.History, error)
+		// - runID can be default(empty string). if empty string then it will pick the last running execution of that workflow ID.
+		// - whether use long poll for tracking new events
```

Comment:
changed to:

// GetWorkflowHistory gets the point in time snapshot of history of a particular workflow
// - workflow ID of the workflow.
// - runID can be default(empty string). if empty string then it will pick the last running execution of that workflow ID.
// - whether use long poll for tracking new events: when the workflow is running, there can be new events generated during iteration
// 	 of HistoryEventIterator, if isLongPoll == true, then iterator will do long poll, tracking new history event, i.e. the iteration
//   will not be finished until workflow is finished; if isLongPoll == false, then iterator will only return current history events.
// - whether return all history events or just the last event, which contains the workflow execution end result
// Example:-
//	To iterate all events,
//		iter := GetWorkflowHistory(ctx, workflowID, runID, isLongPoll, filterType)
//		events := []*shared.HistoryEvent{}
//		for iter.HasNext() {
//			event, err := iter.Next()
//			if err != nil {
//				return err
//			}
//			events = append(events, event)
//		}